test/integ/quote-gouda.test.ts

```diff
@@ -222,6 +224,8 @@ describe('quoteUniswapX', function () {
       parseAmount('5000', UNI_MAINNET),
       parseAmount('4000', WETH9[1]),
       parseAmount('5000000', DAI_MAINNET),
+      parseAmount('18000', agEUR_MAINNET),
+      parseAmount('475000', XSGD_MAINNET),
```

Comment:
did we make sure the UnsiwapX integ tests support these new tokens as well?

Reply:
it should support it but we should just make sure that the carveout portion logic works as expected for classic and uniswapx